core/templates/components/question-directives/question-misconception-editor/question-misconception-editor.component.ts

```diff
@@ -71,9 +74,6 @@ export class QuestionMisconceptionEditorComponent implements OnInit {
   ) {}
 
   ngOnInit(): void {
-    this.misconceptionName = null;
-    this.selectedMisconception = null;
-    this.selectedMisconceptionSkillId = null;
```

Comment:
Check flow, no need to assign null. All 3 is used in misconception-selector, which is used after tagAnswerGroupWithMisconception (where we assign values to all 3). Before that misconception-selector is un-accessable.